processing/src/main/java/io/druid/segment/SimpleQueryableIndex.java

```diff
@@ -63,8 +68,20 @@ public SimpleQueryableIndex(
         columnNamesBuilder.add(column);
       }
     }
+    String[] dimArr = new String[dimNames.size()];
+    int index = 0;
+    for (String dim : dimNames) {
+      if (Strings.isNullOrEmpty(dim)) {
+        log.warn("Null or Empty Dimension found");
```

Comment:
This log line is not necessary, since IndexIO is logging it. And the IndexIO log is more useful since it includes the file name.

Reply:
Removed